authors/apps/articles/utils/model_helpers.py

```diff
@@ -9,3 +9,16 @@ def get_single_article_using_slug(slug):
         return obj
     except Article.DoesNotExist:
         return None
+
+
+def get_all_available_tags():
+    all_tags = set()
+    try:
```

Comment:
this function is too complex, can it be simplified

Reply:
I am going to work on this feedback and reduce complexity in the next 5 minutes @missvicki.
Thanks again for the feedback.